include/cute/atom/copy_traits_xe.hpp

```diff
@@ -1688,6 +1706,60 @@ struct Copy_Traits_<XE_2D_U16x16x16_LD_T, args_t...>
       : XE_2D_LD_Unpack<XE_2D_U16x16x16_LD_T, args_t...>(args...) {}
 };
 
+template <class... args_t>
+struct Copy_Traits_<XE_2D_U8x16x32_LD_T, args_t...>
+    : XE_2D_LD_Unpack<XE_2D_U8x16x32_LD_T, args_t...> {
+  using ThrID = Layout<_16>;
+  // Map from (src-thr,src-val) to bit
+  using SrcLayout = Layout<Shape <_16,_16>,
+                           Stride< _0, _1>>;
+  // Map from (dst-thr,dst-val) to bit
+  using DstLayout = Layout<Shape < _16,Shape <_16,_16>>,
+                           Stride<_256,Stride< _1,_16>>>;
```

Comment:
```suggestion
  using SrcLayout = Layout<Shape <_16,Shape <_8,_32>>,
                           Stride< _0, Stride<_1, _8>>>;
  // Map from (dst-thr,dst-val) to bit
  using DstLayout = Layout<Shape < _16,Shape <_8,_32>>,
                           Stride<_256,Stride< _1,_8>>>;
```

These layouts are wrong. This would be the correct ones, I think.